src/runtime/sgx/untrusted/sgx_module.cc

```diff
@@ -177,10 +198,20 @@ void tvm_ocall_packed_func(const char* name,
 
 // Allocates space for return values. The returned pointer is only valid between
 // successive calls to `tvm_ocall_reserve_space`.
-void* tvm_ocall_reserve_space(size_t num_bytes) {
-  static thread_local std::vector<uint64_t> buf;
-  buf.reserve(num_bytes);
-  return buf.data();
+void* tvm_ocall_reserve_space(size_t num_bytes, size_t alignment) {
+  static TVMContext ctx = { kDLCPU, 0 };
+  static thread_local void* buf;
+  static thread_local size_t buf_size;
+  static thread_local size_t buf_align;
```

Comment:
initialize this

Reply:
`size_t` is scalar and defaults to zero, which is desired. The pointer type init isn't defined, though, so I'll fix that.